5.auth-service/services/userServices.js

```diff
@@ -0,0 +1,197 @@
+const User=require('../models/User');
+const mysql = require('mysql');
+const md5 =require('md5');
+var con = mysql.createConnection({
+    host: "localhost",
+    user: "root",
+    password: "",
+    database: "game_station"
+  });
+  
+  con.connect(function(err) {
+    if (err) throw err;
+    console.log("Connected!");
+  
+  });
+
+
+
+
+
+/**
+ * @param  {} {first_name
```

Comment:
learn how to do JSDoc documentation 
and to that to all functions across the project.